src/client/game.cpp

```diff
@@ -3093,10 +3093,10 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud)
 			!runData.btn_down_for_dig,
 			camera_offset);
 
-	if (pointed != runData.pointed_old) {
+	if (pointed != runData.pointed_old)
 		infostream << "Pointing at " << pointed.dump() << std::endl;
-		hud->updateSelectionMesh(camera_offset);
-	}
+
+	hud->updateSelectionMesh(camera_offset);
```

Comment:
```suggestion
	// Note that updating the selection mesh every frame is not particularly efficient,
	// but the halo rendering code is already inefficient which "cancels" this out.
	hud->updateSelectionMesh(camera_offset);
```
or similar

Reply:
what about "so there's no point in optimizing it here"?

Reply:
thats ok too